shared/networking/2.0.0.md

```diff
@@ -45,7 +45,34 @@ psk=super_secret_wifi_password
 
 This file sets up a simple wifi connection to a network named `My Awesome Wifi Ssid` with a wifi password (`psk`) of `super_secret_wifi_password`. If you want to add multiple different wifi credentials, then simply make a copy of `resin-wifi` or `resin-sample` and change the `SSID` and `psk` key values. You can also add the `autoconnect-priority` integer values in each file to define the priority of each connection when the device has multiple wifi connections it can connect to.
 
-The connection file also defines what kind of security the network expects, if you want to setup a network with a more elaborate security requirements, checkout the [NetworkManager settings reference page][nm-setting-ref].
+To set wifi as the default route to the internet, you need to force `eth0` to have a low route metric and set the `never-default` option. The following ethernet configuration file can be put in the `/system-connections/` folder to accomplish this:
+
+```
+[connection]
+id=my-ethernet
+type=ethernet
+interface-name=eth0
```

Comment:
I probably should be noted that not all ethernet ports will be named in this way. On newer x86 devices they have some weird naming. So it would be recommended that a user first check their ethernet interface name using something like `ifconfig` from the webterminal.